dev-lang/perl/patches/perl-5.40.0.patchset

```diff
@@ -90,6 +90,8 @@ index fa8ebe5..0f09f53 100644
 -cc="gcc"
 -ld='gcc'
 -
+-ccflags="$ccflags -fno-stack-protector"
+-
```

Comment:
This is cleared out together with the rest of this file as before, and later added back in (see below). Should this be merged together?